src/status_im/ui/components/toolbar/view.cljs

```diff
@@ -143,12 +143,14 @@
          content-item]
 
         :else
-        [react/view {:position        :absolute
-                     :left            56
-                     :right           56
-                     :height          styles/toolbar-height
-                     :justify-content :center
-                     :align-items     :center}
+        [react/view
+         (merge {:position        :absolute
+                 :left            16
```

Comment:
`if nav-item 56 16` ? why merge?

Reply:
my-profile does not have `nav-item` like the `back arrow button` in the toolbar, elsewhere there can be

Reply:
i mean instead of `merge` we could just have `:left  (if nav-item 56 16)`

Reply:
oh right 😅 
but still will test it out that it does not affect other toolbars

I think sometimes toolbar does not have `nav-item` but does have "title" as `content-view` and this will cause the "title" to be off-center.. which is not desirable